Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -155,6 +177,32 @@ click: function () { closeClicked.raiseEvent(this); }');
         },
 
         /**
+         * Gets or sets the security settings for iFrames generated by the Infobox.
+         * This will disable sandboxing for the iFrame, and allow HTML to be in the title.
+         * Only do this if you are in complete control of the data that will be loaded into the InfoBox,
+         * otherwise it is a security issue.
+         * @memberof InfoBox.prototype
+         *
+         * @type {Boolean}
+         */
+        disableSecurity : {
```

Comment:
So then the user would disable security by calling something like

    viewer.infoBox.viewer.securityDisabled = false;

correct?